### PR TITLE
test package on julia 1.1

### PR DESCRIPTION
**Pull request by jebej**
Tests are failing on 1.0, for now add testing on 1.1 only

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ os:
 
 julia:
   - 1
+  - 1.0
```
Comment (aviatesk) [inline .travis.yml]: Let's change these lines into a single `- 1.0` since currently `1` is only an alias for `1.3`.

**Comment by jebej** [comment]
Ok there are some deeper problems on 1.0. I'll have to install that version locally to see if I can fix them.

**Comment by aviatesk** [comment]
> Ok there are some deeper problems on 1.0. I'll have to install that version locally to see if I can fix them.

yeah, very much appreciated.
The current error is caused at this line:
https://github.com/JunoLab/Atom.jl/blob/22866b55dfa41848bb1f2242dea0c071d8636397/src/docs.jl#L102

I introduced this branching since `stdlibs` is only defined ≥v1.4, but maybe v1.0 doesn't even have `stdlib`. If you find any alternative for this in `v1.0`, it would solve this problem.

As for return type inference for autocomplete, it may be difficult to find a problem since it very much relies on the internal changes, so I'm okay with to leave it broken.

**Comment by jebej** [comment]
If I manage to fix this I'll make another PR. Maybe the package should be lower bound to 1.1 in the meantime.

**Comment by aviatesk** [comment]
> If I manage to fix this I'll make another PR

Okay, looking forward to the PR !

**Pull request merged**